app/views/base/layout.scala

```diff
@@ -399,18 +396,19 @@ object layout:
       header(id := "top")(
         div(cls := "site-title-nav")(
           !ctx.isAppealUser option topnavToggle,
-          h1(cls := "site-title")(
+          a(cls := "site-title", href := langHref("/"))(
             if ctx.kid.yes then span(title := trans.kidMode.txt(), cls := "kiddo")(":)")
             else ctx.isBot option botImage,
-            a(href := langHref("/"))(siteNameFrag)
+            div(cls := "site-icon", dataIcon := licon.Horsey),
+            div(cls := "site-name")(siteNameFrag)
           ),
-          ctx.blind option h2("Navigation"),
           !ctx.isAppealUser option frag(
             topnav(),
-            ctx.kid.no && ctx.me.exists(!_.isPatron) && !zenable option a(cls := "site-title-nav__donate")(
-              href := routes.Plan.index
+            ctx.kid.no && !ctx.me.exists(_.isPatron) && !zenable option a(cls := "site-title-nav__donate")(
+              href := (if ctx.me.isEmpty then routes.Auth.login else routes.Plan.index)
```

Comment:
shouldn't this always link to /patron, where people are asked to login to actually donate